src/js/modules/breakpointHelper.js

```diff
@@ -60,3 +62,49 @@ export const getCurrentScreenWidth = () => {
     // If no breakpoints have been set
     return false;
 };
+
+export const withinBreakpoint = breakpointString => {
+    const operatorRegex = /[<>=]+/;
+    const operatorMatch = breakpointString.match(operatorRegex);
+    const operator = operatorMatch ? operatorMatch[0] : '';
+    const [, breakpoint] = breakpointString.split(operatorRegex);
+    const currentScreenWidth = window.innerWidth;
+
+    // We fire getCurrentScreenWidth to set the bp array we loop through to get the PX values
+    getCurrentScreenWidth();
```

Comment:
Simialr to comment above – should pull out this functionality so you don't have to call the whole `getCurrentScreenWidth` function just to get the breakpoints array required here.